### DISCUSSION TITLE
Dank Anime Memes

### DISCUSSION THREAD

**Discussion by Unthrottled** [waifu-motivator/waifu-motivator-plugin#286] [Ideas]
## Preface

So I've been really enjoying the interactions that this plugin gives me.
Don't get me wrong, I like anime Waifus, however, I feel like I need a balanced diet of anime content.

I would feel weird adding some cute Imouto as content.
![imouto](https://amii-assets.s3.amazonaws.com/visuals/waiting/waiting_reaction_six.gif)
Because the branding of the plugin is targeted towards Waifu. I don't want the line to be blurred :sweat_smile: 

I am an otaku, so I have other anime interests. I think it would be fun to have something other than anime girls too!

Things like giant robots!

![Eva unit 00](https://media1.tenor.com/images/990f929b844216a630c490deaf904649/tenor.gif?itemid=4934613)

Even content that features male characters as well (not just husbandos). Like my dude Genos

![Genos](https://media1.tenor.com/images/ff03a858f8820187d3a6eb12b7f64494/tenor.gif?itemid=16051370)

It also doesn't have to be limited to characters. 
I also think it would be cool to include Anime openings as well

![Anime Opening](https://media1.tenor.com/images/1cf73d477b24b8138539eca326a3c739/tenor.gif)

## Proposed solution

So I'm currently in the process of building out the [anime meme IDE integration](https://github.com/Unthrottled/AMII) that is solely targeted to delivering a wide variety dank anime-memes. It is purpose was to enable the expansion of the amount of content that could be delivered to the busy programmer. 

I was wondering if we could replace the current meme-based reaction functionality of this plugin by requiring AMII as a required installed plugin. That way the user can control what content they want to see and also be able to more things!

**Comment by cmmata** [comment]
Yes! Giant robots! That's the #1 missing feature I wanted in this plugin 😂  Waifu characters are fun, but sometimes you need something more aggressive :)
